### DISCUSSION TITLE
Firebase doesn't show website instead shows documentation

### DISCUSSION THREAD

**Discussion by Turmaxx** [nunocoracao/blowfish#800] [Q&A] [answered]
my homepage is near complete and is currently on [netlify](tubular-cranachan-7c4e1b.netlify.app/) temporarilty, I just wanted to implement the views and likes feature that comes with blowfish, but I have been running into an issues for the past couple of days. I have read the documentation page [Blowfish: Firebase views and likes](https://blowfish.page/docs/firebase-views/) and followed the steps there to get the project going both on the firebase website and locally.

Essentially the website doesn't come up instead its the documentation does
![image](https://github.com/nunocoracao/blowfish/assets/88465473/979ee6e5-33c2-449a-9573-ff9c7a2debb5)

Here what my project folder looks like.
![image](https://github.com/nunocoracao/blowfish/assets/88465473/46fef10b-8a9b-4172-b025-a7dbb5b0061f)

Here is what my firebase.json file looks like
![image](https://github.com/nunocoracao/blowfish/assets/88465473/3ed6fc35-ed0e-4cbe-b849-ae589073b67d)

**Comment by nunocoracao** [comment]
Hello @Turmaxx  converted this into a discussion as it is not a bug. 
Can you clarify how you are running everything? and also if you have a repo I can take a look at I can try to help when I have time.
If you want examples I would suggest that you look at this repo within the exampleSite folder or to my own personal website for ideas on how the config looks like.

**Comment by Turmaxx** [comment, marked as answer]
Hey @nunocoracao, thanks for looking at this issue. After looking at the [documentation](https://blowfish.page/docs/firebase-views/) slowly. It seems that I have skipped/rushed through a part which was essential for the views and likes to function, it was suppose to be copied onto the fire store database. Code can be found below.

```
rules_version = '2';
service cloud.firestore {
  match /databases/{database}/documents {
    match /{document=**} {
      allow read, write: if request.auth != null;
    }
  }
}
```
- [x] The Views and Likes functionality works.

Regarding the site not being deployed onto Firebase, I have been able to serve it locally but **not been able to deploy it to firebase itself.**

> Can you clarify how you are running everything?

I have `hugo` and `firebase` locally installed on my system previously but they were out of date, so I went ahead and used your `.devcontainer` and `.docker` found on [your website's repo](https://github.com/nunocoracao/nunocoracao.com)

you can look at my [repo](https://github.com/turmaxx/hompage) if you want to take a look

Again Thank you for looking at this issue, I just have a bad habit of rushing through documentation 😅.

I will be doing a pull request to add my site on  https://blowfish.page/users/ after I have my domains publish.
Also thinking about adding a devcontainer or docker container for ease development on the exampleSite.

Again I would like to thank you @nunocoracao , i love the blowfish. and I gave a you a follow. follow back would be great.

**Comment by nunocoracao** [comment]
Check the firebase docs if you're having problems with the ones in Blowfish. Using the firebase CLI should be as easy as 1 command.
